experimental/components/gradle/verification-metadata.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It fails on Windows. I pushed a new `xml` generated by `gradle --write-verification-metadata pgp,sha256 help` suggested in the output. Does it work on macOs?